src/content/developers/tutorials/run-node-raspberry-pi/index.md

```diff
@@ -1,264 +1,153 @@
-
-### 7. Get console output {#7-get-console-output}
-
-You can see what’s happening in the background by typing:
-
-```bash
-sudo tail -f /var/log/syslog
+```shell
+sudo systemctl start geth-lh
+sudo systemctl start lh-geth-beacon
 ```
 
-**Congratulations. You are now running a full Ethereum node on your Raspberry Pi 4.**
-
-## Syncing the Blockchain {#syncing-the-blockchain}
-
-Now you need to wait for the blockchain to be synced. In the case of the execution layer this will take a few days depending on several factors but you can expect up to about 5-7 days.
-
-If you are running the consensus layer Prater testnet you can expect 1-2 days of Beacon chain synchronization time. Remember that you will need to setup the validator later in order to start the staking process. [How to run the consensus layer validator](/developers/tutorials/run-node-raspberry-pi/#validator)
+The logs can be checked using
 
-## Monitoring dashboards {#monitoring-dashboards}
-
-For this first release, we included 3 monitoring dashboards based on Prometheus [[5]](/developers/tutorials/run-node-raspberry-pi/#references) / Grafana [[6]](/developers/tutorials/run-node-raspberry-pi/#references) in order to monitor the node and clients’ data (Geth and Besu). You can access through your web browser:
-
-```bash
-URL: http://your_raspberrypi_IP:3000
-User: admin
-Password: ethereum
+```shell
+# logs for Geth
+sudo journalctl -u geth-lh -f
+#logs for lighthouse
+sudo journalctl -u lh-geth-beacon -f
 ```
 
-## Switching clients {#switching-clients}
+The specific service names for every combination of clients are available at the [Ethereum on Arm docs](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/kiln/kiln-testnet.html#id2). Theyu can be used to update the commands provided here for any combination.
 
-All clients run as a systemd service. This is important because if a problem arises the system will respawn the process automatically.
+## Validators {#validators}
 
-Geth and Prysm beacon chain run by default (depending on what you are synchronizing, execution layer or consensus layer) so, if you want to switch to other clients (from Geth to Nethermind, for instance), you need to stop and disable Geth first, and enable and start the other client:
+In order to run a validator it is first necessary to deposit 32 test-ether into the Kiln deposit contract. This means the user must have access to an account containing 32 ETH. This can be done by following the step-by-step guide on the [Kiln Launchpad](https://kiln.launchpad.ethereum.org/en/). Do this on a desktop/laptop, built do not generate keys - this can be done directly on the Raspberry Pi.
```

Comment:
> This means the user must have access to an account containing 32 ETH. This can be done by following the step-by-step guide on the [Kiln Launchpad](https://kiln.launchpad.ethereum.org/en/)

This kinda sounds like there is a faucet on the Launchpad. 

```suggestion
In order to run a validator you must first have access to 32 testnet ETH, which must be deposited into the Kiln deposit contract. This can be done by following the step-by-step guide on the [Kiln Launchpad](https://kiln.launchpad.ethereum.org/en/). Do this on a desktop/laptop, but do not generate keys—this can be done directly on the Raspberry Pi.
```